docs/spec/data-plane.md

```diff
@@ -91,6 +91,14 @@ delivered.
 CloudEvents received by Sink MAY have
 [Distributed Tracing Extension Attribute](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md).
 
+### Event reply header
+
+An event sender, including Source and Broker and Channel, SHOULD include
```

Comment:
Not sure if we should keep the original name "Forwarding entity" as from the proposal instead of sender. Any thoughts?

Reply:
It is not the event sender, as I ready the original proposal, it is scoped to Knative middleware like a Channel or Broker, but not something like a Source.

Reply:
Good point. But I think sender and receiver is more concepts of endpoints, which is not tied to Source or Sink. eg, in the source side, sender will be source and the receiver will be broker, while in the Sink side sender will be the broker and receiver will be the Sink. So sender can refer to all source, channel and broker.